plugins/package-managers/python/src/main/kotlin/Pip.kt

```diff
@@ -87,6 +93,14 @@ class Pip(
         }
     }
 
+    private val analyzeSetupPyInsecurelyOption = options[OPTION_ANALYZE_SETUP_PY_INSECURELY]?.also { analyzeSetupPy ->
+        require(analyzeSetupPy in BOOLEAN_VALUES) {
```

Comment:
As it seems a bit odd to have the `BOOLEAN_VALUES` list, I wonder whether we should instead do e.g.

        requireNotNull(analyzeSetupPy.toBooleanStrictOrNull()) {
            "The '$OPTION_ANALYZE_SETUP_PY_INSECURELY' option must be 'true' or 'false', but was '$analyzeSetupPy'."
        }

Reply:
That's definitely better. Even better would be if it was possible to use actual booleans in the config file.

Reply:
> if it was possible to use actual booleans in the config file.

True, plugin-specific typed options are not supported, as it's a bit unclear how plugin should declare their expected types...